assets/less/night/content/general.less

```diff
@@ -0,0 +1,10 @@
+h1, h2, h3, h4, h5, h6 {
```

Comment:
Just to be consistent with the rest of the code, please, use one line per header:

```css
h1,
h2,
h3,
h4,
h5,
h6 {
...
}
```